ESPController/web_src/default.htm

```diff
@@ -682,25 +682,37 @@ <h1 id="v0">Victron CANBUS integration</h1>
     </div>
 
     <div class="page" id="diybmsCurrentMonitorPage">
-        <h1>diyBMS Current &amp; Voltage Monitor</h1>
+        <h1>Current &amp; Voltage Monitoring</h1>
 
         <div class="region">
             <h2>Connection</h2>
             <p id="b1">Configure the MODBUS connection to the current monitor using the settings below.</p>
+            <p id="b14">PZEM-017 device uses 9600,8,None,2 serial settings</p>
             <form id="diybmsCurrentMonitorForm1" method="POST" action="savecurrentmon.json" autocomplete="off">
                 <div class="settings">
                     <div>
                         <label for="CurrentMonEnabled">Enabled</label>
                         <input type="checkbox" name="CurrentMonEnabled" id="CurrentMonEnabled">
                     </div>
+                    <div>
+                        <label for="CurrentMonDev">Device</label>
+                        <select name="CurrentMonDev" id="CurrentMonDev">
+                            <option value="0">DIYBMS Current Monitor</option>
+                            <option value="1">PeaceFair PZEM-017</option>
```

Comment:
Perhaps an `onChange` action can be registered such that if the user selects the PZEM-017 other parameters will automatically be updated to the required defaults?